Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift

```diff
@@ -23,7 +23,7 @@ public final class HyperwalletPrepaidCard: HyperwalletTransferMethod {
     override private init(data: [String: AnyCodable]) {
         super.init(data: data)
     }
-
+    /// Initialization of HyperwalletPrepaidCard
```

Comment:
update to - /// The required initializer

Reply:
Updated